common/tools/turborepo-remote-cache/src/index.ts

```diff
@@ -32,20 +32,27 @@ fastify.register(authenticateToken);
 fastify.register(authenticateTeamId);
 
 const port = (process.env.AZURE_CACHE_PORT || 3000) as number;
-const azureStorageAccount = process.env.AZURE_STORAGE_ACCOUNT;
-const azureStorageContainerName = process.env.AZURE_STORAGE_CONTAINER_NAME;
 const packageVersion = process.env.PACKAGE_VERSION || "1.0.0";
 
-if (!azureStorageAccount || !azureStorageContainerName) {
-  fastify.log.error("Missing Azure Storage account or container name.");
-  process.exit(1);
-}
+let containerClient: ContainerClient;
 
-const blobServiceClient = new BlobServiceClient(
-  `https://${azureStorageAccount}.blob.core.windows.net`,
-  new DefaultAzureCredential(),
-);
-const containerClient = blobServiceClient.getContainerClient(azureStorageContainerName);
+if (process.env.AZURE_CONTAINER_SAS_URL) {
+  containerClient = new ContainerClient(process.env.AZURE_CONTAINER_SAS_URL);
+} else {
```

Comment:
Shouldn't we have an else if which fails that says nothing is configured?

Reply:
The old code already checks for AZURE_STORAGE_ACCOUNT and AZURE_STORAGE_CONTAINER_NAME. It is moved into the else block below.